packages/create-pantheon-decoupled-kit/src/templates/next-drupal-app/lib/constants.tsx

```diff
@@ -0,0 +1,6 @@
+export const DRUPAL_URL = process.env.backendUrl || 'undefined';
+export const IMAGE_URL = process.env.imageUrl || DRUPAL_URL;
+export const BUILD_MODE =
+	process.env.npm_lifecycle_script?.includes('next build');
+export const LOCALES = process.env.locales as unknown as string[];
```

Comment:
I don't think we need this anymore, The docs say to fetch the valid locales in the middleware. Not sure that example I sent is doing that either, but we should be able to repurpose the `get-locales` script into the middleware and do away with setting this var at all. In other words, right now we are fetching locales at build time and they are statically available to the app at run time. With the app dir, we should be able to fetch the locale per-request (and I believe the response will be cached with the app dir magic), then the locale can be given to the renderer on the server which spits out the correctly translated content.